cromwellApiClient/src/main/scala/cromwell/api/CromwellClient.scala

```diff
@@ -29,12 +29,21 @@ class CromwellClient(val cromwellUrl: URL, val apiVersion: String)(implicit acto
   def abortEndpoint(workflowId: WorkflowId): String = workflowSpecificEndpoint(workflowId, "abort")
   def statusEndpoint(workflowId: WorkflowId): String = workflowSpecificEndpoint(workflowId, "status")
   def metadataEndpoint(workflowId: WorkflowId): String = workflowSpecificEndpoint(workflowId, "metadata")
+  def outputsEndpoint(workflowId: WorkflowId): String = workflowSpecificEndpoint(workflowId, "outputs")
+  def logsEndpoint(workflowId: WorkflowId): String = workflowSpecificEndpoint(workflowId, "logs")
+  def diffEndpoint(workflowA: WorkflowId, callA: String, indexA: ShardIndex, workflowB: WorkflowId, callB: String, indexB: ShardIndex): String = {
+    def shardParam(aOrB: String, s: ShardIndex) = s.index.map(i => s"&index$aOrB=$i.toString").getOrElse("")
```

Comment:
Security wise, I think it's the Cromwell service's job to make sure "Bobby Tables" URLs don't work (otherwise anyone could craft them, regardless of this library)

So, usability wise, do you think it'd be better to:
- Tell people that their request will be rejected upfront, rather than letting Cromwell decide?
- Let Cromwell decide, and let people get their errors from Cromwell rather than the client?